x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/rules_tables.tsx

```diff
@@ -294,10 +299,8 @@ export const RulesTables = React.memo<RulesTableProps>(
     ]);
 
     useEffect(() => {
-      if (reFetchRulesData != null) {
-        setRefreshRulesData(reFetchRulesData);
-      }
-    }, [reFetchRulesData, setRefreshRulesData]);
+      setRefreshRulesData(reFetchRules);
```

Comment:
Would we also update line 93 in this file to reflect the new typing of `refetchRules`?

Reply:
Oh yea. Good catch there.

Reply:
I did a few more searches and found 4 more uses of that boolean in signatures I'm removing.